web/migrations/0052_add_meme_slug.py

```
@@ -0,0 +1,39 @@
+# Generated by Django 5.1.6 on 2025-04-01 16:15
+
+from django.db import migrations, models
+from django.utils.text import slugify
+
+
+def populate_meme_slugs(apps, schema_editor) -> None:
```

Ruff (0.8.2)

Missing type annotation for function argument `apps`

(ANN001)